openssl-dynamic/src/main/c/ssl.c

```diff
@@ -721,19 +656,11 @@ TCN_IMPLEMENT_CALL(jint, SSL, initialize)(TCN_STDARGS, jstring engine)
     if (r) {
```

Comment:
(not introduce by this PR, but while we are here) `r` is an integer type ... lets use it like an integer: `r != 0`